InternshipProject#2/Helpers/GenerateToken.cs

```diff
@@ -0,0 +1,55 @@
+<<<<<<<< HEAD:InternshipProject#2/Helpers/GenerateToken.cs
+using InternshipProject_2.Models;
+========
+﻿using InternshipProject_2.Models;
+>>>>>>>> master:InternshipProject#2/Helpers/TokenGenerator.cs
```

Comment:
what is this ????
you didn't resolve the conflicts....

Reply:
fixed conflict